src/main/java/it/gov/pagopa/connector/onetrust/OneTrustRestServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package it.gov.pagopa.connector.onetrust;
+
+import it.gov.pagopa.dto.onetrust.PrivacyNoticesDTO;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Service
+@Slf4j
+public class OneTrustRestServiceImpl implements OneTrustRestService {
+
+    @Autowired
+    OneTrustRestClient oneTrustClient;
+
+    /**
+     * Calls OneTrust privacy notices' service passing {@link LocalDateTime#now()} as parameter
+     */
+    @Override
+    public PrivacyNoticesDTO getPrivacyNotices(String id) {
+        return this.getPrivacyNotices(id, LocalDateTime.now());
+    }
+
+    /**
+     * Calls OneTrust privacy notices' service to retrieve the versions less than the given date-time and the highest version
+     */
+    @Override
+    public PrivacyNoticesDTO getPrivacyNotices(String id, LocalDateTime date) {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-ddTHH:mm:ss");
```

Comment:
the DateTimeFormatter is an immutable thread safe utility. We could handle it as a constant in order  to create it just once